simplq/src/api/requestFactory/index.js

```diff
@@ -0,0 +1,4 @@
+import * as QueueRequestFactory from './queue';
+import * as TokenRequestFactory from './token';
+
+export { QueueRequestFactory, TokenRequestFactory };
```

Comment:
Could you change the name from QueueRequestFactory to something else, maybe just QueueRequests? Since it's not a factory, the names just feels out of place and I think it'll confuse people looking at the code.

Reply:
It's a request factory. It's creating request objects. An example:

```
export const create = (queueName) => ({ method: 'post', url: '/queue', data: { queueName } });
```

It takes a queue name and creates a queue creation request object.